interfaces/cython/cantera/examples/thermo/equivalenceRatio.py

```diff
@@ -2,52 +2,116 @@
 This example demonstrates how to set a mixture according to equivalence ratio
 and mixture fraction.
 
-Requires: cantera >= 2.5.0
+Requires: cantera >= 2.6.0
 """
 
 import cantera as ct
 
 gas = ct.Solution('gri30.yaml')
 
-# fuel and oxidizer compositions
-fuel = "CH4"
-oxidizer = "O2:0.21,N2:0.79"
+# Set the mixture composition according to the stoichiometric mixture
+# (equivalence ratio phi = 1). The fuel composition in this example
+# is set to 100 mol-% CH4 and the oxidizer to 21 mol-% O2 and
+# 79 mol-% N2.
+gas.set_equivalence_ratio(phi=1.0, fuel="CH4:1", oxidizer="O2:0.21,N2:0.79")
 
-gas.TP = 300, ct.one_atm
+# The following is a short hand notation and equivalent to the call above
+gas.set_equivalence_ratio(1.0, "CH4", "O2:0.21,N2:0.79")
```

Comment:
I believe just using the 'long' notation (i.e. keyword arguments) throughout this example will be easier to follow (as it is self-documenting). It still makes sense to document that fuel can be `CH4` or `CH4:1`